web-service/helpers.py

```diff
@@ -132,42 +130,48 @@ def SortByDistanceCmp(a, b):
 
 ################################################################################
 
-def FetchAndStoreUrl(siteInfo, url):
+def GetSiteInfoForUrl(url, rssi=None, txpower=None, force_update=False):
+    siteInfo = models.SiteInformation.get_by_id(url)
```

Comment:
In most of the case, people are forced to use URL shorteners to fit their URLs in the BLE advertising packet and anyway, they would probably like to get fast response for their beacon data.

If you're not associating any cached data to the short URL, in 90% of the case, the cache won't be used, which is questioning the purpose of the cache at all.